packages/ui/src/components/Designer/RightSidebar/DetailView/index.tsx

```diff
@@ -75,21 +75,34 @@ const DetailView = (
     form.setValues(values);
   }, [form, activeSchema]);
 
-  const handleWatch = (newSchema: any) => {
+  const handleWatch = (formSchema: any) => {
+    const formAndSchemaValuesDiffer = (formValue: any, schemaValue: any): boolean => {
+      if (typeof formValue === 'object') {
+        return JSON.stringify(formValue) !== JSON.stringify(schemaValue);
```

Comment:
this is needed otherwise comparisons of objects like `dynamicFontSize` would always be false and therefore always considered as changed.